src/components/color_picker/color_picker.ts

```diff
@@ -195,6 +195,7 @@ export interface ColorPickerProps {
   dropdownDirection?: "left" | "right" | "center";
   onColorPicked: (color: Color) => void;
   currentColor: Color;
+  hideNoColorButton?: boolean;
```

Comment:
nitpick: A better name would probably be `disableNoColor` of `disableEmptyColor`. Something that does not explicitely mention on button :) what do you think ?